tests/integration/targets/mso_schema_template_contract_filter/tasks/main.yml

```diff
@@ -549,6 +498,7 @@
     - nm_change_filter_directives_log_and_none_pc.previous.directives[0] == "no_stats"
     - nm_change_filter_directives_log_and_none_pc.current.directives == ["log", "none", "no_stats"]
 
+# change contract scope
```

Comment:
Trivial comment, not needed. Same is written in name below.

Reply:
ok